### ISSUE TITLE
Change default branch to "main"

### ISSUE THREAD

**Issue by vincepri** [kubernetes-sigs/kubebuilder#1583]
Once the community agrees on the naming, we need to change our default development branch to `main`, and check that any tools or references to it are changed as well.

This includes the container image tag used in development.

/kind cleanup
/wg naming

**Comment by camilamacedo86** [comment]
We will not longer use the master branch?

**Comment by vincepri** [comment]
Yes, it's not something we can do immediately. I've filed this issue in multiple repositories to track the work

cc @justaugustus

**Comment by mirogta** [comment]
See https://www.techrepublic.com/article/github-to-replace-master-with-main-starting-in-october-what-developers-need-to-know/

**Comment by vitorfloriano** [comment]
Hi @camilamacedo86 I was wondering what it would take to get this issue closed.

Do we only need to rename all the references to the `master` branch to `main` and then change the settings on git and GitHub (default branch, branch protection rules, etc)?

Would we need to ping the org admins to change the settings after we rename all the references?

What about the open PRs against `master`?

**Comment by camilamacedo86** [comment]
Thanks for raising this.

We support inclusive naming, and for new projects we already default to `main`. For this existing repo, though, renaming the default branch from `master` is a high-risk change: it is referenced in many places (docs, CI/release automation, website/Netlify, external guides, forks, downstream tooling), and not all of those updates can be validated or coordinated from here (I also don’t ever have access to update every required system).

Because the risk of unintended breakage is significant and we currently don’t have the access/coordination needed to do this safely, I’m going to close this issue for now.

If someone would like to lead the work in the future, it would need a full mapping of all `master` references and a coordinated migration plan (including docs + website/Netlify + automation + downstream impact). Please comment here or ping the maintainers if you want to take that on.
